src/core/components/load-element-error/index.tsx

```diff
@@ -19,16 +19,24 @@ const LoadElementError = (props: Props) => {
 
   return (
     <SafeAreaView style={styles.safeArea}>
-      <TouchableOpacity onPress={props.onPressReload}>
-        <View style={styles.container}>
-          <View style={styles.textWrapper}>
-            <Text style={styles.title}>{getError()}</Text>
-          </View>
+      <View style={styles.container}>
+        <View style={styles.textWrapper}>
+          <Text style={styles.title}>{getError()}</Text>
+        </View>
+        <TouchableOpacity onPress={props.onPressReload}>
```

Comment:
Maybe we could make both the error message and the reload button touchable since the earlier implementation was doing that and a better experience for the user as they don't need to target the button accurately while trying to reload.

Reply:
I merged as is to get it shipped. I'm happy to go over updates with you if you'd like. One concern with having the touch area go beyond the verbs was when the text wraps to multiple I wanted to make sure it didn't overlap with the other button.